packages/catalog-process/src/routers/EServiceRouter.ts

```diff
@@ -366,18 +366,22 @@ const eservicesRouter = (
 
         try {
           // The same check is done in the backend-for-frontend, if you change this check, change it there too
-          validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE]);
+          validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE, M2M_ADMIN_ROLE]);
```

Comment:
The comment above is not true anymore, since `M2M_ADMIN_ROLE` is not present in the `backend-for-frontend` check. 

I would consider finding another fix for the bug solved in https://github.com/pagopa/interop-be-monorepo/pull/2177/files, avoiding this exception in our authorization patterns - for example, catching and rethrowing the 404 into 403 in the process service logic.

We could track this as a further APIV V2 task and just update the comment for the scope of this PR

Reply:
Will be addressed in #2227 or tracked as a new task outside API V2